api-docs/machine-insights-streaming-dataformat.md

```diff
@@ -0,0 +1,305 @@
+    "3-2": "The type of access used for the operation. This can be on of DIGITAL_KEY, ROLLING_PIN, STATIC_PIN, KEY_CARD, KEYPAD_USE",
+    "4-0": "Success",
+    "4-1": "true",
+    "4-2": "Boolean value for whether the operation was successful.",
+    "5-0": "datetime",
+    "5-1": "1604042795000",
+    "5-2": "[Unix timestamp](https://en.wikipedia.org/wiki/Unix_time) in milliseconds, for when this operation was recorded."
+  },
+  "cols": 3,
+  "rows": 6,
+  "align": [
+    "left",
+    "left",
+    "left"
+  ]
+}
+[/block]
+
+## Hours: CumulativeOperatingHours, CumulativeIdleHours, ...
+
+In general, the field CumulativeOperatingHours is the most interesting: this is the total amount of hours for the machine. Other fields might be present as well, if those datapoints are available for this machine, such as CumulativeIdleHours, CumulativeIdleNonOperatingHours, CumulativeProductiveHours, CumulativeMovingHours, etc. The general format of the Hours fields is:
+
+| Field    | Example       | Description                                                                                                      |
+|:---------|:--------------|:-----------------------------------------------------------------------------------------------------------------|
+| Hour     | 42.12         | Decimal amount of hours.                                                                                         |
+| datetime | 1604042795000 | [Unix timestamp](https://en.wikipedia.org/wiki/Unix_time) in milliseconds, for when this datapoint was recorded. |
+
+## Machine Insights
+
+The main fields (around 80 in total) all describe some aspect of semantic data about a machine: FuelRemaining, EngineStatus, Distance travelled, etc. The availability of these datapoints will vary depending on the source of the telematic data (be it GPS, CAN-bus, ISO feed, etc.). The unit of measurement for each datapoint will always be the same, i.e., the unit for Distance will always be "km" for kilometres. The general format of the machine insights is, with varying field names:
+
+[block:parameters]
+{
+"data": {
+"h-0": "Field",
+"h-1": "Example",
+"h-2": "Description",
+"0-0": "Field with value (naming varies)",
+"0-1": "13.8  \ntrue",
+"0-2": "The value of the datapoint itself. Floating point or boolean.",
+"1-0": "unit (naming sometimes varies)",
+"1-1": "\"km\"  \n\"volt\"",
+"1-2": "The unit of measurement for this dataseries. Will always stay the same for the same field.",
+"2-0": "datetime",
+"2-1": "1604042795000",
+"2-2": "[Unix timestamp](https://en.wikipedia.org/wiki/Unix_time) in milliseconds, for when this datapoint was recorded."
+},
+"cols": 3,
+"rows": 3,
+"align": [
+"left",
+"left",
+"left"
+]
+}
+[/block]
+
+For the exact semantics and availability of these datapoints, please see the [data model description.](https://dev.trackunit.com/docs/trackunit-data-model)
+
+## CANMessages
+
+> 📘 This is probably not the data you are looking for
```

Comment:
Not sure about that :)

Reply:
That's just copy-paste from what we already have published on: https://developers.trackunit.com/reference/machine-insights-streaming-dataformat 😜 

I will look at your suggestions later. I want to publish new insights before Bartek sends the email.